cli/tools/init/mod.rs

```diff
@@ -40,6 +40,15 @@ pub async fn init_project(init_flags: InitFlags) -> Result<(), AnyError> {
     .replace("{CURRENT_STD_URL}", deno_std::CURRENT_STD_URL.as_str());
   create_file(&dir, "main_test.ts", &main_test_ts)?;
 
+  create_file(&dir, "deno.json", include_str!("./templates/deno.json"))?;
+  let vscode_dir = dir.join(".vscode");
+  std::fs::create_dir_all(vscode_dir)?;
+  create_file(
+    &vscode_dir,
+    "settings.json",
+    include_str!("./templates/settings.json"),
+  )?;
```

Comment:
Perhaps we should only do this if we can find vscode on the path? (ex. maybe just something like `env("PATH").contains("Microsoft VS Code")`.. not sure what that would be on linux/mac)

Reply:
My personal opinion is to just add it, I think its very important in deno's case because every vscode user has to setup deno, also
- Most ppl have vscode installed even if its not they're main editor
- Even in neovim I'm planing to use an extension that parses vscode settings and apply the common deno part

Reply:
neovim ecosystem now have https://github.com/folke/neoconf.nvim which makes .vscode/settings.json just works in nvim, I think this is another argument in favor of adding this

Reply:
Sure, that sounds good to just add it.